notebooks/samples/explanations/ai-explanations-tabnet-algorithm.ipynb

```diff
@@ -0,0 +1,540 @@
+{
```

Comment:
<p>Change section title to "Reading a sample TabNet prediction" or something similar.</p><p> </p><p>"After training and serving your model, you upload the output to Cloud Storage. There is a sample TabNet prediction using synthetic data: </p><p>"</p><p> </p><p>Also explain a bit about aggregated mask values here - what are they, and what are we using them for?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/182/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>